### ISSUE TITLE
Getting Error in Adding this project as a dependency

### ISSUE THREAD

**Issue by kishanios123** [ish-app/ish#2021]
I created new demo project and added this project as a dependency ...

when try to build project i am getting many errors, attaching screenshot here ...

Please let me know if any other information needed ...

![Screenshot 2023-01-17 at 7 12 50 PM](https://user-images.githubusercontent.com/60137209/212914664-5937ec02-d20e-4367-a37b-591140f7d2d1.png)
![Screenshot 2023-01-17 at 7 13 14 PM](https://user-images.githubusercontent.com/60137209/212914645-534083ce-2a90-4fb1-b44b-6d7960fef5f2.png)

**Comment by saagarjha** [comment]
Can you share more details about what you are trying to do?

**Comment by kishanios123** [comment]
> Can you share more details about what you are trying to do?

I want to use ish Terminal commands in a new project, and try to add this project as a dependency in a new project.

Can someone mention steps to do this or can you make Cocoapod or something like it for easy integration?

**Comment by saagarjha** [comment]
iSH is modular, but it depends on what you need it to do. There's a core for the interpreter and runtime that is shared across iOS and Linux, and there's also iOS-specific UI code. Which parts are you looking to use?

**Comment by kishanios123** [comment]
> iSH is modular, but it depends on what you need it to do. There's a core for the interpreter and runtime that is shared across iOS and Linux, and there's also iOS-specific UI code. Which parts are you looking to use?

I want to use commands in my app, so I want to integrate functional parts only... The UI part is not needed ... I want to execute python commands from my app...

**Comment by saagarjha** [comment]
If you just want to run Python code in your app, I think it would be a better idea to embed a Python interpreter such as CPython?

**Comment by kishanios123** [comment]
@saagarjha thanks for your suggestion, but I also want to execute some other dependency available in ish ... I really appreciate this project and your effort ... I will be happy if you can provide a solution for my problem.

**Comment by saagarjha** [comment]
The easiest way to make this work is probably to add the iSH Xcode project to yours and link against libish.

**Comment by kishanios123** [comment]
> The easiest way to make this work is probably to add the iSH Xcode project to yours and link against libish.

Yes, I have tried the same but getting errors as shown in the attached images in my first message...

**Comment by saagarjha** [comment]
Do you have a project you can share?

**Comment by kishanios123** [comment]
> Do you have a project you can share?

I will send you by mail.

**Comment by kishanios123** [comment]
> If you just want to run Python code in your app, I think it would be a better idea to embed a Python interpreter such as CPython?

Can you suggest to me some libraries for iOS to execute python commands?
